source/server/http/admin.cc

```diff
@@ -542,46 +542,53 @@ Http::Code AdminImpl::handlerCerts(absl::string_view, Http::HeaderMap&,
 
 Http::Code AdminImpl::handlerRuntime(absl::string_view url, Http::HeaderMap& response_headers,
                                      Buffer::Instance& response) {
-  Http::Code rc = Http::Code::OK;
   const Http::Utility::QueryParams params = Http::Utility::parseQueryString(url);
-  const auto& entries = server_.runtime().snapshot().getAll();
-  const auto pairs = sortedRuntime(entries);
+  response_headers.insertContentType().value().setReference(
+      Http::Headers::get().ContentTypeValues.Json);
 
-  if (params.size() == 0) {
-    for (const auto& entry : pairs) {
-      response.add(fmt::format("{}: {}\n", entry.first, entry.second.string_value_));
-    }
-  } else {
-    if (params.begin()->first == "format" && params.begin()->second == "json") {
-      response_headers.insertContentType().value().setReference(
-          Http::Headers::get().ContentTypeValues.Json);
-      response.add(runtimeAsJson(pairs));
-      response.add("\n");
-    } else {
-      response.add("usage: /runtime?format=json\n");
-      rc = Http::Code::BadRequest;
+  rapidjson::Document document;
```

Comment:
Can you add a TODO here to protoize this? I should be able to get to this soon when I finish up the other config_dump stuff.